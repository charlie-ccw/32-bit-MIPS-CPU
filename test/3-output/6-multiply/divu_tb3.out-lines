register_v0_final 3ffffffc
