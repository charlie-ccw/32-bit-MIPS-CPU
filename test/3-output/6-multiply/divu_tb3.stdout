RAM : INIT : Loading RAM contents from test/1-binary/6-multiply/divu_tb3.txt
TB : INFO : RAM_ACCESS: Read from 0xbfc00000, data: 0x2403fff2
TB : INFO : RAM_ACCESS: Read from 0xbfc00004, data: 0x24020004
TB : INFO : RAM_ACCESS: Read from 0xbfc00008, data: 0x0062001b
TB : INFO : RAM_ACCESS: Read from 0xbfc0000c, data: 0x00001012
TB : INFO : RAM_ACCESS: Read from 0xbfc00010, data: 0x00000008
TB : INFO : RAM_ACCESS: Read from 0xbfc00014, data: 0x00000000
TB : INFO : RAM_ACCESS: Read from 0x00000000, data: 0x00000000
register_v0_final 3ffffffc
test/bus_cpu_tb.v:52: $finish called at 430 (1s)
